BeeJet.Tests/AddGameCommandHandlerTests.cs

```diff
@@ -17,7 +17,7 @@ public async Task AddGameAsync_WithoutAdminUser_SendsAddAdminRoleMessage()
             var guild = GuildFixture.GuildWithAdminRole;
 
             var service = Substitute.For<IGDBService>(string.Empty, string.Empty);
-            var commandHandler = new AddGameCommandHandler(service);
+            var commandHandler = new AddGameCommandHandler(service, null);
```

Comment:
Wil je hier niet gewoon 2 ctors ? 1 met alleen Service en 1 met Service en DB? 
Want DB is blijkbaar optioneel en veelal leeg. Nu moet je overal null meegeven en je weet niet wat "null" is

Reply:
Default parameter gemaakt met 0, kan niet 2 contstructors doen vind de depency injection niet zo leuk